extensions/authentication-jwt/src/__tests__/fixtures/controllers/user.controller.ts

```diff
@@ -45,8 +57,43 @@ export class UserController {
     public userService: UserService<User, Credentials>,
     @inject(SecurityBindings.USER, {optional: true})
     private user: UserProfile,
+    @inject(UserServiceBindings.USER_REPOSITORY)
+    public userRepository: UserRepository
   ) {}
 
+  @post('/users/signup', {
+    responses: {
+      '200': {
+        description: 'User Signup route',
+        content: {
+          'application/json': {
+            schema: {
+              'x-ts-type': User,
+            },
+          },
+        },
+      },
+    },
+  })
+  async signUp(
+    @requestBody({
+      content: {
+        'application/json': {
+          schema: CredentialsSchema,
+        },
+      },
+    })
+    newUserRequest: NewUserRequest,
+  ): Promise<User> {
+    const password = await hash(newUserRequest.password, await genSalt());
+    delete newUserRequest.password;
+    var savedUser = await this.userRepository.create(newUserRequest);
```

Comment:
```suggestion
    const savedUser = await this.userRepository.create(newUserRequest);
```

Reply:
I make the changes you suggested please check now.